dask_ml/dask_ml_simple.py

```diff
@@ -8,9 +8,9 @@
 
 """
 
-import dask.array as da
 import optuna
 
+import dask.array as da
```

Comment:
This change is to avoid an `isort` error in the `dask-ml` example, though I'm not sure why this didn't appear over in https://github.com/optuna/optuna-examples/pull/45